src/app/app.module.ts

```diff
@@ -3,14 +3,23 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import {AuthModule} from "./auth/auth.module";
```

Comment:
same same import format in all imports, now some has space between and some doesn't

```
import { AppComponent } from './app.component';
             ^^ have space
import {AuthModule} from "./auth/auth.module";
             ^ don't have space
```